src/server/server.js

```diff
@@ -134,6 +140,15 @@ app.get(
   },
 );
 
+app.post(
+  '/lti/article-iframe/:lang/article/:articleId',
```

Comment:
If it is because topic-articles is'nt supported we should refactor  '/article-iframe/:lang/:resourceId/:articleId' to support it and reuse that endpoint instead of adding a new one.